examples/general/ami/post_installation.md

```diff
@@ -0,0 +1,127 @@
+
+# Install EFA Driver(only required for multi-instance training)
+curl -O https://efa-installer.amazonaws.com/aws-efa-installer-latest.tar.gz
+wget https://efa-installer.amazonaws.com/aws-efa-installer.key && gpg --import aws-efa-installer.key
+cat aws-efa-installer.key | gpg --fingerprint
+wget https://efa-installer.amazonaws.com/aws-efa-installer-latest.tar.gz.sig && gpg --verify ./aws-efa-installer-latest.tar.gz.sig
+tar -xvf aws-efa-installer-latest.tar.gz
+cd aws-efa-installer && sudo bash efa_installer.sh --yes
+cd
+sudo rm -rf aws-efa-installer-latest.tar.gz aws-efa-installer
+
+# Remove pre-installed package and Install Neuron Tools
+sudo yum remove aws-neuron-tools  -y
+sudo yum remove aws-neuronx-tools  -y
+sudo yum install aws-neuronx-tools-2.*  -y
+```
+
+2. Change `install_neuron.sh` to executable:
+```
+chmod +x install_neuron.sh
+```
+
+3. Create a slurm script with the following content and name it as `install_neuron.slurm`:
+
+```
+#!/bin/bash
+#SBATCH --nodes=16
+#SBATCH --exclusive
+srun ./install_neuron.sh
+```
+
+and run it:
+
+```
+sbacth install_neuron.slurm
+```
+
+After this is completed, all compute nodes will have necessary Neuron packages installed.
+
+4. Set up Python virtual environment in head node. In this step, create a shell script with the following content and name it as `install_python_env.sh`:
+
+```
+# Configure Linux for Neuron repository updates
+
+sudo tee /etc/yum.repos.d/neuron.repo > /dev/null <<EOF
+[neuron]
+name=Neuron YUM Repository
+baseurl=https://yum.repos.neuron.amazonaws.com
+enabled=1
+metadata_expire=0
+EOF
+sudo rpm --import https://yum.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB
+
+# Install OS headers
+sudo yum install kernel-devel-$(uname -r) kernel-headers-$(uname -r) -y
+
+# Update OS packages
+sudo yum update -y
+
+# Remove pre-installed package and Install Neuron Tools
+
+sudo yum remove aws-neuron-tools  -y
+sudo yum remove aws-neuronx-tools  -y
+sudo yum install aws-neuronx-tools-2.*  -y
+
+# Install Python venv and activate Python virtual environment to install Neuron pip packages.
+python3.7 -m venv aws_neuron_venv_pytorch_p37
+source aws_neuron_venv_pytorch_p37/bin/activate
+python -m pip install -U pip
+
+# Install packages from beta repos
+
+python -m pip config set global.extra-index-url "https://pip.repos.neuron.amazonaws.com"
+# Install Python packages - Transformers package is needed for BERT
+python -m pip install torch-neuronx=="1.11.0.1.*" "neuronx-cc==2.*" transformers
+```
+
+5. Change `install_python_env.sh` to executable:
+```
+chmod +x install_python_env.sh
+```
+and run `install_python_env.sh` in head node terminal:
+```
+sh install_python_env.sh
+```
+This is run as a shell script instead of Slurm script. This is because we only want to create the virtual environment once, which is in the head node only. 
+
+After this step is complete, the ParallelCluster is properly configured to run SLURM jobs.
+
+
+## Helpful SLURM commands
```

Comment:
Let's discuss later how to incorporate this section https://quip-amazon.com/lLDYAbCHlyTb/How-to-use-parallel-cluster-head-node-to-launch-SLURM-jobs-on-trn132xlarge-worker-nodes#temp:C:CXR04731f4373b340efa23087794 into the tutorial.

Reply:
How about a link here to a new md that includes the relevant how-to section